src/utils/network_info.ts

```diff
@@ -810,6 +835,7 @@ export const cosmosNetworkKeys = [
   "ssv",
   "stafihub",
   "stargaze",
+  "stargaze-testnet",
```

Comment:
Seems forgot to remove these

Reply:
🦅

Actually both of them 1783b42b0e6f883e20f5688fb85512ddb4ac80ab 🙈